maxim/logger/logger.py

```
@@ -836,14 +837,16 @@ def tool_call_update(self, tool_call_id: str, data: Dict[str, Any]):
         """
         ToolCall.update_(self.writer, tool_call_id, data)
 
-    def tool_call_result(self, tool_call_id: str, result: Any):
+    def tool_call_result(self, tool_call_id: str, result: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `tool_call_result`

Add return type annotation: `None`

(ANN201)